src/server/src/main/java/com/spotify/reaper/resources/view/NodesStatus.java

```diff
@@ -66,75 +86,114 @@ private GossipInfo parseEndpointStatesString(String sourceNode, String allEndpoi
       Optional<String> releaseVersion = Optional.absent();
       Optional<String> tokens = Optional.absent();
       Optional<Double> load = Optional.absent();
-      
+
       String endpointString = strEndpoints[i];
-      
-      matcher = endpointNamePattern.matcher(endpointString);
-      if(matcher.find()) {
-        endpoint = Optional.fromNullable(matcher.group(1));
+
+      for (Pattern endpointNamePattern : endpointNamePatterns) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "endpointNamePattern" which hides the field declared at line 42. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)